boards/aludel_elixir_v1_ns_B.overlay

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright (c) 2024 Golioth, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&qwiic_i2c {
```

Comment:
Nope! I didn't realize that it was possible to just have a single `aludel_elixir_v1_ns.overlay` that would be picked up for all board revisions. I've updated the PR to just use that single file.

Reply:
> How are the different revisions of the boards selected?

@szczys you can explicitly specify the revision by adding a `@<REV>` to the board identifier:

```sh
# Rev A
west build -p -b aludel_elixir_v1_ns@A

# Rev B
west build -p -b aludel_elixir_v1_ns@B
```

If you leave off the revision specifier, it will build the [default board defined in revision.cmake](https://github.com/golioth/golioth-zephyr-boards/blob/045a9c391fcc25dd6d16883ed9b87b4e14489e61/boards/arm/aludel_elixir_v1/revision.cmake#L7) (currently defaults to Rev B):

```sh
west build -p -b aludel_elixir_v1_ns
```